test/welcomePage.spec.js

```diff
@@ -0,0 +1,21 @@
+import { shallowMount, mount } from "@vue/test-utils"
+import AppBody from "../components/AppBody.vue"
+import { BJumbotron } from 'bootstrap-vue'
+
+describe('Welcome page', () => {
+  it('Welcome page should display what you can create', () => {
+    const wrapper = shallowMount(AppBody)
+    expect(wrapper.find("p").text()).toBe("You can create new one or see and edit your current employees")
```

Comment:
If I change this text directly inside component (AppBody), and run this single test, then I can see weird error, not related to the expected output (that something is wrong with text):
Error: `Unknown custom element: <b-jumbotron> - did you register the component correctly?`
So seems like test didn't reach `expect(...)` line yet, that's why error is different.
To solve this issue, you need to tell the file with tests about all other components dependecies that you use in component.
`const wrapper = shallowMount(AppBody, {
      stubs: {
        BJumbotron
      }
    });`

I would also move this to `BeforeAll` part to render component with all dependecies BEFORE all tests are executed and do this only once.
Then try to change this line with text in component and run test - you should see finally proper error message, that compared texts are not the same :)